packages/taquito-sapling/src/sapling-tx-builder/sapling-transactions-builder.ts

```diff
@@ -23,9 +23,11 @@ import { convertValueToBigNumber } from '../sapling-tx-viewer/helpers';
 import { SaplingState } from '../sapling-state/sapling-state';
 import { SaplingWrapper } from '../sapling-module-wrapper';
 import { InMemorySpendingKey } from '../sapling-keys/in-memory-spending-key';
+import { InMemoryProvingKey } from '../sapling-keys/in-memory-proving-key';
 
 export class SaplingTransactionBuilder {
   #inMemorySpendingKey: InMemorySpendingKey;
+  #inMemoryProvingKey: InMemoryProvingKey | undefined;
```

Comment:
does this also work as `#inMemoryProvingKey?: InMemoryProvingKey` ?

Reply:
yes